config/application.yml.template

```diff
@@ -1,51 +0,0 @@
-# Add configuration values here, as shown below.
```

Comment:
Yes but that's the template that other devs need to set up locally. It has all the ENV variables they have to set in `application.yml` file.